src/config.js

```diff
@@ -28,8 +28,9 @@ const config = {
         useYou: true
       },
       realtimeActive: false,
+      omitCommentBubbles: true,
```

Comment:
@kadmil We have these:
- https://dev.freefeed.net/T38
- https://dev.freefeed.net/T39